pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.pkgproj

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+
+  <PropertyGroup>
+    <Version>2.0.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <File Include="files\Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.props">
```

Comment:
from where would those (UWPShim & msbuild taks/targets ) come for very-short term. Are they coming from msi. So you would have both msi & nuget package for now. How would this .props file & .targets file become part of the app build.